src/contrib/mlir/tools/ngraph-opt/ngraph_opt.cpp

```diff
@@ -61,6 +61,10 @@ static llvm::cl::opt<bool>
                   llvm::cl::desc("Run the verifier after each transformation pass"),
                   llvm::cl::init(true));
 
+static llvm::cl::opt<bool> allowUnregisteredDialects("allow-unregistered-dialects",
```

Comment:
Do you need a CL for this ? I would hard-code it to false below, unless there is a reason.

Reply:
Will hard-code.